src/Makefile

```diff
@@ -0,0 +1,124 @@
+## Haiku Generic Makefile ##
+
+## Fill in this file to specify the project being created, and the referenced
+## Makefile-Engine will do all of the hard work for you. This handles any
+## architecture of Haiku.
+
+# The name of the binary.
+NAME = DeeperPeople
+
+# The type of binary, must be one of:
+#	APP:	Application
+#	SHARED:	Shared library or add-on
+#	STATIC:	Static library archive
+#	DRIVER: Kernel driver
+TYPE = APP
+
+# 	If you plan to use localization, specify the application's MIME signature.
+APP_MIME_SIG = 
```

Comment:
Can you add the signature application/x.vnd.DeepOcean-PEPL please?

Reply:
Ah, the signature is in PeopleApp.h. Thanks!